extensions/kotlin/runtime/src/main/kotlin/org/apache/camel/quarkus/kotlin/RoutesDSL.kt

```diff
@@ -14,14 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.acme.timer
+package org.apache.camel.quarkus.kotlin
 
+import org.apache.camel.RoutesBuilder
 import org.apache.camel.builder.RouteBuilder
 
-class TimerRoute: RouteBuilder() {
-    override fun configure() {
-        from("timer:foo?period=1s")
-                .process { e -> e.message.body = "Hello from Kotlin!" }
-                .log("\${body}")
+fun routes(block: RouteBuilder.() -> Unit) : RoutesBuilder {
```

Comment:
Could you please add a piece of KDoc explaining what is the code in this file supposed to do?

Reply:
it's fairly obvious but I'll add some doc

Reply:
> it's fairly obvious but I'll add some doc

Do not laugh at us - Kotlin noobs!